### DISCUSSION TITLE
Howto expand ui on full space of view (in code)?

### DISCUSSION THREAD

**Discussion by Alexdnk** [lllyasviel/Fooocus#3619] [Q&A] [answered]
I've looked through all the code, but I haven't found a constant or variable that is responsible for the size of these fields on the sides. I want to remove them completely. How to do it? Does anyone know where in the code it is set?
![Снимок экрана 2024-09-13 162522](https://github.com/user-attachments/assets/5bcd1413-aca5-4053-b3d3-3a0c2632e5fb)

**Comment by Yogesh-DevHub** [comment, marked as answer]
in css folder -> style.css add this at the end 
@media (min-width: 1536px) {
  .app.svelte-1kyws56.svelte-1kyws56 {
      /* max-width: 1536px; */
      max-width: 100%;
  }
}

**Reply by Alexdnk** [reply]
Thanks you.
